Parsers/ASimDns/Parsers/ASimDnsInfobloxBloxOne.yaml

```diff
@@ -0,0 +1,227 @@
+Parser:
```

Comment:
are DNS EventTypes mentioned in the following page supported in this parser?
https://docs.infoblox.com/space/BloxOneThreatDefense/35438586/DNS+Security+Policy+Hit+and+RPZ+Hit+Log+Message+Mapping

Reply:
Yes, The DNS EventTypes mentioned in the following page supported in this parser.

Reply:
do we have any sample log for that eventtype? if not can you attach?

I'm not sure if Security Policy related will pass through this condition. As it seem it won't contain DNS

![image](https://github.com/user-attachments/assets/b41bb1a0-cfab-401d-884d-d675d0ee22fe)

Reply:
I apologise for the confusion earlier, the DNS EventTypes mentioned in this page https://docs.infoblox.com/space/BloxOneThreatDefense/35406922/DNS+Query%252FResponse+Log+Message+Mapping are supported in the parser, for which we have submitted the sample logs.

Reply:
I understand, is it possible to add support for the other eventtype as well. The other one seems more critical as it may include threat related events?

Reply:
At the moment, we have limited sample data available, and the DNS Security Policy Hit and RPZ Hit Log are not included in our current scope of work.